runner/modal_app.py

```diff
@@ -164,6 +164,24 @@ def text_to_image_sdxl_lightning_8step_api():
     return make_api("text-to-image", "ByteDance/SDXL-Lightning-8step")
 
 
+@stub.function(image=api_image, secrets=[Secret.from_name("api-auth-token")])
```

Comment:
The Starter plan for Modal only supports 8 deployed endpoints so as-is I think deploying this app would fail. Assuming that is the case, I would recommend removing the following endpoints to stay under the limit:

- `image_to_video_svd_api()` -> can just use the SVD1.1 endpoint

Reply:
You're right. That's a good catch. I noticed that error yesterday when I deployed a test endpoint to my modal account. I have now commented on the SVD1.0 endpoint in 7334b3afadd80a8f97a4d1437490bfd4f80ed650 👍🏻.